neuralop/models/uno.py

```diff
@@ -195,7 +201,7 @@ def __init__(self,
                                             use_mlp=use_mlp, 
                                             mlp_dropout=mlp_dropout, 
                                             mlp_expansion=mlp_expansion,
-                                            output_scaling_factor = self.uno_scalings[i],
+                                            output_scaling_factor = [self.uno_scalings[i]],
```

Comment:
Why is it a list?

Reply:
The Fno blocks takes scaling_factors which are 2d list
Scaling factor for each layer and scaling factor along each dims. 
It can also take a scaler and also 1D list with scaling factor along each layer.